test/interactions/keyInteractionTests.ts

```diff
@@ -75,7 +77,7 @@ describe("Interactions", () => {
         svg.remove();
       });
```

Comment:
The "can remove keyPress callbacks" should assert that the function returns the calling object.